gmusicapi/clients/mobileclient.py

```diff
@@ -43,6 +43,14 @@ def _ensure_device_id(self, device_id=None):
 
         return device_id
 
+    def _validate_device_id(self, device_id):
+        """Ensure that a given device_id belongs to the user supplying it."""
+        if device_id in [d['id'][2:] if d['id'].startswith('0x') else d['id']
+                         for d in self.get_registered_devices()]:
+            return device_id
+        else:
+            raise ValueError('Invalid device_id.')
```

Comment:
Thoughts on adding some advice on resolving this problem in the exception (or maybe the login docstring)? I guess we could actually output the possible device ids to pick from in the message, even.